staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -554,17 +554,17 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			versionedObject = defaultVersionedObject
 		}
 		reqScope.Namer = action.Namer
-		namespaced := ""
+		namespaced := "Cluster"
```

Comment:
There was a race with my last response, sorry. Yes I'll make the change accordingly. Thanks.